src/Data/Characters/Shenhe/index.tsx

```diff
@@ -156,7 +156,7 @@ export const data = dataObjForCharacterSheet(key, elementKey, "liyue", data_gen,
   },
   teamBuff: {
     premod: {
-      all_dmgInc: nodeSkill,
+      cryo_elemental_dmgInc: nodeSkill,
```

Comment:
```suggestion
      all_dmgInc: equal(input.hit.ele, "cryo", nodeSkill),
```

Tried this out, but wasn't working for me

Reply:
Looks like there is a bug where hit specific entries like `ele/crit/base` cant be read behind `teambuff`. should be fixed soonTM